src/sidebar/components/Excerpt.js

```diff
@@ -86,23 +98,27 @@ function Excerpt({
   const [collapsedByInlineControls, setCollapsedByInlineControls] =
     useState(true);
 
-  // Container for the excerpt's content.
   const contentElement = /** @type {{ current: HTMLDivElement }} */ (useRef());
 
-  // Measured height of `contentElement` in pixels.
+  // Measured height of `contentElement` in pixels
   const [contentHeight, setContentHeight] = useState(0);
 
-  // Update the measured height of the content after the initial render and
-  // when the size of the content element changes.
-  const updateContentHeight = useCallback(() => {
-    const newContentHeight = contentElement.current.clientHeight;
-    setContentHeight(newContentHeight);
-
-    // prettier-ignore
-    const isCollapsible =
+  const updateContentHeight = useCallback(
+    /**
+     * Update the measured height of the content container after initial render,
+     * and when the size of the content element changes.
+     */
```

Comment:
The _description_ of a function is best attached to the `const updateContentHeight` variable. Only the parameter types needed to be attached to the wrapped closure. I'd also favor an ordinary comment here as using JSDoc notation just uses extra space and this is not an external API that someone would try to use from another module.